src/actions/TrackingActions.ts

```diff
@@ -14,13 +14,13 @@ export function trackConversion(
     pluginId: string
     orderId?: string
   }
-): ThunkAction<Promise<unknown>> {
-  return async (dispatch, getState) => {
+): ThunkAction<unknown> {
```

Comment:
While we are in here, should this be `ThunkAction<void>`? Seems weird to return `unknown`.